src/stories/_exec/__init__.py

```diff
@@ -3,24 +3,28 @@
 
 try:
     from inspect import iscoroutinefunction
-except SyntaxError:
+except ImportError:
     iscoroutinefunction = lambda x: False
```

Comment:
It'll be syntax error on python 2 I guess. We should handle both.

Reply:
```
Python 2.7.16 (default, Jul  4 2019, 19:49:40) 
[GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from inspect import iscoroutinefunction
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name iscoroutinefunction
```

Reply:
I strongly suggest to use `asyncio.iscoroutinefunction` instead.